app/pages/3_Country_operation.py

```diff
@@ -105,7 +105,7 @@ def get_colors_map():
 with suppl_col:
     choices = upd_dict
     res = st.selectbox(
-        "Resolution",
+        "Country",
```

Comment:
As we discussed, that may be important to have functionality to use time-aggregation as a kind of smoothing filter. So, I think `Resolution` box should be back.